jepsen/src/jepsen/control/util.clj

```diff
@@ -102,22 +104,30 @@
   of the filename, downloading a new version could silently give you the old
   version instead.
 
-  Options:
-
-    :force?     Even if we have this cached, download the tarball again anyway."
+  Params:
+    user        user for wget authentication. If provided, valid pw must also be provided.
+    pw          password for wget authentication.
+    force?      Even if we have this cached, download the tarball again anyway."
   ([url]
-   (wget! url {:force? false}))
-  ([url opts]
```

Comment:
Here, I think it'd make more sense to take an option map rather than having n layers of optional arguments.

Reply:
Will do!

Reply:
They are now all part of the option map. What do you think?